src/firmware/app/primitives/move_manager.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <stdbool.h>
+
+#include "firmware/app/world/firmware_world.h"
+#include "shared/proto/primitive.nanopb.h"
+
+/**
+ * Start moving in the given world
+ * @param void_state_ptr [in] A pointer to this primitives state, as allocated by the
+ *                            `create_state` function
+ * @param world [in] The world this primitive is running in.
+ * @param move_position_params Move Position Params
+ * @param final_angle The final angle
+ */
+void app_move_manager_start(void *void_state_ptr, FirmwareWorld_t *world,
+                            MovePositionParams move_position_params, float final_angle);
```

Comment:
Why not call this "move_primitive" instead? That's effectively what this is, no?

Reply:
I think it'll be nuked and replaced by trajectory follower or something in https://github.com/UBC-Thunderbots/Software/issues/1654, so I'll leave it as move_helper for now